frontend/packages/console-shared/src/utils/user-settings.ts

```diff
@@ -58,11 +58,14 @@ export const updateConfigMap = async (configMap: ConfigMapKind, key: string, val
     ];
     try {
       await k8sPatch(ConfigMapModel, configMap, patch);
+      return Promise.resolve();
     } catch (err) {
       // eslint-disable-next-line no-console
       console.error(err);
+      return Promise.reject();
```

Comment:
This is an async function. You should be throwing errors and returning values instead of returning promises.

Reply:
same as above

Reply:
I will fix this in my PR. 😄